from launch import LaunchDescription
from  launch_ros.actions import Node

def generate_launch_description():
    return LaunchDescription([
        Node(
            package='vff_avoidance',
            executable='avoidance_vff_node',
            output='screen',
            parameters=[{
                'use_sim_time': True
            }],     
            remappings=[
                ('input_scan', '/scan'),
                ('output_vel', '/cmd_vel')
            ]
        )
    ])

   